src/translations/en.json

```
@@ -2641,6 +2641,13 @@
               "name": "Entity",
               "description": "The Entity card gives you a quick overview of your entity’s state."
             },
+            "entity_filter": {
+              "card": "Card",
+              "delete_state": "Delete state",
+              "entities": "Entities",
+              "show_empty": "Display when empty",
```

```suggestion
              "show_empty": "Display card when empty",
```
Makes it clearer what is shown/hidden.